cluster/prod/knative/BUILD.bazel

```diff
@@ -0,0 +1,64 @@
+# Copyright 2021 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+load("@io_bazel_rules_k8s//k8s:object.bzl", "k8s_object")
+load("@io_bazel_rules_k8s//k8s:objects.bzl", "k8s_objects")
+
+k8s_objects(
+    name = "prod",
+    objects = [
+        ":namespace",  # Must be first
+        ":summarizer",
+        ":updater",
+    ],
+)
+
+CLUSTER = "{STABLE_TESTGRID_CLUSTER}"
+
+MULTI_KIND = None
+
+k8s_object(
+    name = "updater",
```

Comment:
Will you please:

a) Move these targets into //cluster/prod/BUILD.bazel
b) Use `template = "knative/updater.yaml" and `name = "knative-updater"` (etc)
c) Delete this BUILD.bazel file

Otherwise this won't actually apply as part of the prod rollout (which is what we want)

Reply:
Note: leave the cluster/prod/knative/summarizer.yaml, etc in their current locations, but reference them in cluster/prod/BUILD.bazel

Reply:
good idea, done